packages/router/test/router_preloader.spec.ts

```diff
@@ -196,6 +200,272 @@ describe('RouterPreloader', () => {
            })));
   });
 
+  describe('should support preloading stratages', () => {
+    let delayLoadUnPasued: BehaviorSubject<string[]>;
+    let delayLoadObserver$: Observable<string[]>;
+    let events: Array<RouteConfigLoadStart|RouteConfigLoadEnd>;
+
+    const subLoadChildrenSpy = jasmine.createSpy('submodule');
+    const lazyLoadChildrenSpy = jasmine.createSpy('lazymodule');
+    const buildPreloader = (
+        delayObserver$: Observable<string[]>,
+        ): typeof PreloadingStrategy => {
+      class DelayedPreLoad implements PreloadingStrategy {
+        preload(route: Route, fn: () => Observable<any>): Observable<any> {
+          const routeName =
+              route.loadChildren ? (route.loadChildren as jasmine.Spy).and.identity : 'noChildren';
+          return delayObserver$.pipe(
+              filter(unpauseList => unpauseList.indexOf(routeName) !== -1), take(1),
+              switchMap(() => {
+                return fn().pipe(
+                    catchError(() => of(null)),
+                );
+              }));
```

Comment:
```suggestion
              }),);
```
nit: adding a trailing comma will put each of the operators on a new line when formatted and it'll be a bit easier to read.